qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -306,16 +305,8 @@ def meanfield(
         )
         raise TypeError(error_message)
 
-    package_dir = os.path.join(outpath.strip(), package)
-    basis_dir = os.path.join(package_dir, basis.strip())
-
-    if not os.path.isdir(package_dir):
-        os.mkdir(package_dir)
-        os.mkdir(basis_dir)
-    elif not os.path.isdir(basis_dir):
-        os.mkdir(basis_dir)
-
-    path_to_file = os.path.join(basis_dir, name.strip())
+    filename = name + "_" + package.lower() + "_" + basis.strip()
```

Comment:
Any specific reason for having the basis set in the name, or just to distinguish calculations with different basis sets?

Reply:
Yes, exactly.